tsdb/index/tsi1/index.go

```diff
@@ -707,7 +707,7 @@ func (i *Index) TagSets(name []byte, opt influxql.IteratorOptions) ([]*influxql.
 				}
 			}
 			// Associate the series and filter with the Tagset.
-			tagSet.AddFilter(string(SeriesElemKey(e)), e.Expr())
+			tagSet.AddFilter(string(models.MakeKey(e.Name(), e.Tags())), e.Expr())
```

Comment:
Is this only for read-only operations, because it seems like the behaviour of `SeriesElemKey` and `MakeKey` are quite different? I ask only in terms of backwards compatibility if this result is persisted anywhere.